projects/ui/src/Components/Common/Modal.tsx

```diff
@@ -5,17 +5,24 @@ export function Modal({
   headContent,
   title,
   bodyContent,
+  className,
 }: {
   onClose: () => any;
   headContent: JSX.Element; // ususally just an icon
   title?: string;
   bodyContent?: JSX.Element;
+  className?: string;
 }) {
   // The modal "mask" is overriden in "main.scss" rather than with
   //   the overlayProps so that we have easy access to our
   //   color constants.
   return (
-    <MantineModal opened={true} onClose={onClose} centered>
+    <MantineModal
+      opened={true}
+      onClose={onClose}
+      className={className}
```

Comment:
Does this apply itself to the entire mask+body component, just the body, or some inner piece?  I think a more descriptive name might help, especially as Mantine's documentation on this exact case is sparse and only mentions scrollbars.

Reply:
This is applied outside the modal and page overlay - I'll add that comment.